src/components/common/StaffPayment.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <section id="staff-payment">
     <header>
-      <h2>3. Staff Payment</h2>
+      <h2>4. Staff Payment</h2>
```

Comment:
oh weird that it is picking up a diff here, i believe this change was already in.

Reply:
Ditto.